src/harness/fake_sas.py

```diff
@@ -202,6 +202,30 @@ def GetEscSensorRecord(self, request, ssl_cert=None, ssl_key=None):
       # Return Empty if invalid Id
       return {}
 
+  def GetFullActivityDump(self, version, ssl_cert=None, ssl_key=None):
+    response = json.loads(json.dumps({
+        'files': [
+            {'url': "https://raw.githubusercontent.com/Wireless-Innovation-Forum/\
+             Spectrum-Access-System/master/schema/empty_activity_dump_file.json",
```

Comment:
Alignment: one space in the lines 209, 213,217 & 221 should be removed to align correctly.

Reply:
Alignment corrected.